content/widgets.md

```diff
@@ -66,3 +66,5 @@
 	* Child view controller should be fine
 
 * If you have more than one widget, use the `UIApplicationShortcutWidget` Info-Plist key to specify the widget (by bundle ID) which should be shown upon 3D-touching the homescreen app icon.
+
+* Data Protection should not be enabled for widgets. If you do, iOS may fail to create a snapshot of your widget when showing the widgets in the lock screen. In this case, Springboard will crash the next time your widget should be displayed (as the snapshot is missing).
```

Comment:
Could you specify what that means technically? Is this a capability/info plist key, and which one?